.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -6,6 +6,10 @@ labels: ["bug", "triage"]
 
 ## Bug Report
 
+**Before you file your issue**
+- Check the other [issues](https://github.com/suitenumerique/docs/issues) before filing your own
+- If your issue is related to the text editor ([BlockNote](https://github.com/TypeCellOS/BlockNote)) please [file it on their repo](https://github.com/TypeCellOS/BlockNote/issues). If you don't know file away ! We'll tell you where it belongs or even file it for you if we need it too 😊
```

Comment:
I suggest rephrasing the second item: "If your report is related to the BlockNote text editor, file it on [their repo](https://github.com/TypeCellOS/BlockNote/issues). If you're not sure whether the problem is with BlockNote or Docs, file the bug on our repo: if we support it, we'll backport it upstream ourselves, otherwise we'll ask you to do so."

If you keep the current phrasing, just a nitpick: you can remove the space before the "!".